src/World.cpp

```diff
@@ -3905,15 +3905,20 @@ cFluidSimulator * cWorld::InitializeFluidSimulator(cIniFile & a_IniFile, const c
 	Printf(SimulatorNameKey, "%sSimulator", a_FluidName);
 	AString SimulatorSectionName;
 	Printf(SimulatorSectionName, "%sSimulator", a_FluidName);
-	AString SimulatorName = a_IniFile.GetValueSet("Physics", SimulatorNameKey, "Vanilla");
+
+	bool IsWater = (strcmp(a_FluidName, "Water") == 0);  // Used for defaults
+	
+	AString DefaultSimulatorName = GetDimension() == dimNether && IsWater ? "Vaporise" : "Vanilla";
+	
```

Comment:
What's with all the empty lines? Just bunch the code together, it's one logical block.